engine/src/idraw.cpp

```diff
@@ -317,6 +319,8 @@ void MCImage::drawwithgravity(MCDC *dc, MCRectangle r, MCGravity p_gravity)
             dx = r . x;
             dw = r . width;
             break;
+        default:
```

Comment:
I think this should just be `case kMCGravityNone:` (and the same for the following `switch` block).